gear-contract-libraries/Cargo.lock

```diff
@@ -0,0 +1,429 @@
+# This file is automatically @generated by Cargo.
```

Comment:
No need for `Cargo.lock` for `rlib`. 

https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries